.github/workflows/tox.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
What about 3.12?

Reply:
I feel like I'm already out of line removing 3.8 here.  I'd rather have a separate PR to clean up that business (which is also a separate issue).